### DISCUSSION TITLE
非常感谢作者的辛勤维护，我想可以给手表单独安排一个版本，这样子会非常受欢迎，可以提高项目的知名度

### DISCUSSION THREAD

**Discussion by Firct** [zhihulite/Hydrogen#64] [Ideas]
之前手表其实改一下软件的dpi是可以用的，我的手表为oppo watch 2.但是自从

> 2024/6/7 11:53:07

> new`` 0.517

> 修复开启或关闭夜间模式图标不更新 修复鼠标滚轮无法滚动问题详情 修复回答标题栏内容显示不精确 修复删除收藏夹内容报错 修复想法无法加载视频 新增对于付费回答的加载和标注 新增评论区点击用户头像或名称直接打开主页 新增打开自动滚动到指定位置 去除内置浏览器打开

> [下载](https://download.zoho.com.cn/public/workdrive/download/rnv2n41600fa139f4466594114b4fe0f34981)``

> 2024/5/19` 15:21:52

> new 0.516

> 修复鼠标滚轮无法滑动回答页的bug`

0.517更新以后，回答就加载不出来了，我推测是因为从那一次更新以后，换了pdf的显示方式，但是显然手表并没有相关的库。但是0.516是可以正常使用的，而且还十分好用，我用appcloner将缩放比例跳到了50%，用的很舒服，但是因为老版本的原因，没有办法显示相关用户的所有回答了。
如果能够给出一个特供版，我将感激不尽❤❤❤

**Comment by huajiqaq** [comment]
下个版本可能会修复无法使用的问题 应该是webview版本过低 js代码使用到了当前webview不支持的api

**Comment by huajiqaq** [comment]
经查验 发现是webview版本过低导致的 不打算为老版本webview修复 请自行升级webview
